03/arr-to-bst.md

```diff
@@ -0,0 +1,221 @@
+# Array-to-BST
+
+## Problem Statement
+
+Given a sorted array of integers, `arr`, write a function to create a balanced Binary Search Tree from the contents of the array. Return the root of the Binary Search Tree.
+
+Example:
+
+`arr = [5, 10, 15, 20, 25, 30, 35, 40, 45]`
+
+should result in a tree with the following root/height:
+
+![Balanced Binary Search Tree](../images/balanced_bst.png)
+
+Please note one is not required to implement a self-balancing Binary Search Tree in order to solve this exercise. 
+
+It is recommended to break the problem down recursively by first setting the root of the Binary Search Tree to the middle element of the array.
```

Comment:
Consider moving this to the hints section of the sample pseudocode implmentation